chore: Docs improvements (#979)
temporalio/sdk-typescript: README.md

  </a>
</div>

> [Temporal](https://temporal.io) is a distributed, scalable, durable, and highly available orchestration engine used to execute asynchronous, long-running business logic in a scalable and resilient way.

Reviewer: nit: I would avoid mentioning 'scalable' twice. It fits better as a quality of the engine (ie. early in the sentence) than as a quality of the execution (ie. later in the sentence)

> [Temporal](https://temporal.io) is a distributed, scalable, durable, and highly available orchestration engine used to execute asynchronous, long-running business logic in a scalable and resilient way.

Reviewer 2: ```suggestion
[Temporal](https://temporal.io) is a durable execution system that transparently makes your code durable, fault-tolerant, and simple.
```

Reviewer 2: candace + oxford comma

Author: I was going to update after my discussion on Slack with Candace.
Thanks.

"Temporal TypeScript SDK" is the framework for authoring workflows and activities using either the TypeScript or JavaScript programming languages.

> For getting started and general documentation, please refer to the following resources:

Reviewer: Had to read the "For getting started" part several times before I got to parse the correct meaning of this sentence. Not sure how to best fix, but you can consider this:

```suggestion
For "Getting started instructions" and General documentation, please refer to the following resources:
```

Author: Let me think about this one.

> For getting started and general documentation, please refer to the following resources:

Reviewer 2: ```suggestion
To get started, see:
```

Author: This is nice but the resources are not just for getting started.
